systemd.yaml

```diff
@@ -181,7 +182,8 @@ subpackages:
             -s test-tpm2 \
             -s test-fd-util \
             -s test-fstab-generator.sh \
-            -s test-rm-rf
+            -s test-rm-rf \
+            -s test-execute
```

Comment:
Looks like you need to also skip `test-mount-util` and `test-namespace`

Reply:
I see they're failing because first they need the new microvm-init, I'll explicitly add them

Reply:
Now PR depends on https://github.com/wolfi-dev/os/pull/56962